src/Tizen.NUI/src/internal/Interop/Interop.ActorProperty.cs

```diff
@@ -204,6 +204,9 @@ internal static partial class ActorProperty
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Actor_Property_BLEND_EQUATION_get")]
             public static extern int BlendEquationGet();
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Actor_Property_TOUCH_AREA_get")]
```

Comment:
could you share the csharp-binder's patch?
sometimes, runtime crash comes caused by mismatch of charp-binder version.

Reply:
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/246392/